### PR TITLE
Allow accessType per remote method

### PR DESCRIPTION
**Pull request by raymondfeng**
/to @ritch 

It depends on https://github.com/strongloop/strong-remoting/pull/147.

### CODE REVIEW

File: lib/model.js [hunk 1]
```diff
@@ -314,6 +314,25 @@ Model._getAccessTypeForMethod = function(method) {
 
   var ACL = Model._ACL();
 
+  // Check the explicit setting of accessType
+  if (method.accessType) {
+    assert(method.accessType === ACL.READ ||
```
Comment (ritch) [inline lib/model.js]: Can we avoid hard coding these? I know we have the enum... but that is already a bit static. Custom access types might be nice in the future.

File: lib/model.js [hunk 2]
```diff
@@ -314,6 +314,25 @@ Model._getAccessTypeForMethod = function(method) {
...
+    return method.accessType;
+  }
+
+  // Default GET requests to READ
+  var verb = method.http && method.http.verb;
+  if (typeof verb === 'string') {
+    verb = verb.toUpperCase();
+  }
+  if (verb === 'GET' || verb === 'HEAD') {
+    return ACL.READ;
```
Comment (ritch) [inline lib/model.js]: Maybe we should allow the user to specify this via `config.json`. You could just check `app.get('access').defaultAccessTypes[verb]`.

**Comment by ritch** [comment]
Feel free to ignore my suggestions. LGTM

**Comment by raymondfeng** [comment]
Good suggestions for extensibility of access types. I'll leave the enhancements to future work.

**Pull request merged**